src/utils/guid.test.js

```diff
@@ -1,52 +1,42 @@
 import guid from './guid';
 
 describe('guid', () => {
-  beforeEach(() => {
-    Math.random = jest.fn(() => 0.4677647565236618);
-  });
+  Math.random = jest.fn(() => 0.4677647565236618);
```

Comment:
Will this bleed into a different test suite? Do you need to "undo" this patch after all of these tests have run?

Reply:
Good point. It won't right now but it is safe to add a `clearAllMocks`

Reply:
@biharck, if you use Math.random somewhere else, I think it would still be a jest.fn? Doesn't clearAllMocks just reset the function so you can see what args are passed?

I think you need to do something like...

```
const savedFn = Math.random;
Math.random = jest.fn();
// after all
Math.random = savedFn;
```

Reply:
For example: https://github.com/facebook/jest/issues/7136#issuecomment-428750792

But I think you're right that `Math.random` calls in other test suites would use the baked in implementation, and not the one we override here.

Reply:
For example: https://github.com/facebook/jest/issues/7136#issuecomment-428750792

But I think you're right that `Math.random` calls in other test suites would use the baked in implementation, and not the one we override here.